sections/1-introduction.tex

```diff
@@ -3,28 +3,33 @@ \section{Introduction}
 
 Research astrophysicists rely on software to analyze increasingly large and complex data sets .
 Scientists that study our nearest star, the Sun, are no different.
-Solar physicists rely primarily on remote sensing data from both space- and ground-based observatories to measure the properties of our star and deduce the physical mechanisms at work.
+Solar physicists rely on remote sensing data from both space- and ground-based instruments to measure the properties of our star and deduce the physical mechanisms at work.
 In the past, most solar data analysis was performed using Fortran, a general-purpose, compiled programming language designed for scientific and engineering applications.
-In the 1980s, the solar physics community made the transition from Fortran to the Interactive Data Language (IDL), a commercial and closed-source programming language.
-The community extracted several benefits from this transition.
-IDL is a vector-oriented interpreted programming language which enables much faster and therefore cheaper development compared to Fortran.
-It also comes packaged with a large number of powerful libraries that support scientific data visualization and analysis.
-For similar reasons, the astronomy community also made the transition to IDL which led to the release of the IDL Astronomy User's Library for the solar community to leverage.
-Since then, significant and powerful functionality has been developed for solar data analysis as part of SolarSoftWare, a set of freely-available integrated IDL software libraries that provide a common data analysis environment \citep{Freeland:1998we}.
+Several reasons motivated the solar physics community to adopt the Interactive Data Language (IDL), a commercial and closed-source programming language, in the 1980s.
+For one, IDL is a vector-oriented interpreted programming language which enables much faster and therefore cheaper development compared with Fortran.
+Second, IDL includes many libraries that support scientific data visualization and analysis.
+Finally, the solar community benefitted from functionality developed by the astronomy community (specifically, the IDL Astronomy User's Library).
+Since then, SolarSoftWare (SSW) developed significant functionality for the solar community through a set of freely-available integrated IDL software libraries that provide a common data analysis environment \citep{Freeland:1998we}.
```

Comment:
This reads a bit awkward and makes it sound as if SolarSoftWare is a group of people that developed code, rather than the name of the code.

Reply:
How about:

> Since then, \citep{Freeland:1998we} developed significant functionality for the solar community through a set of freely-available integrated IDL software libraries called SolarSoftWare that provide a common data analysis environment.

Reply:
I think it is missing the gist to say that someone developed SolarSoft _for_ the community. It was the community that developed much of the code distributed within SolarSoft for _itself_. 

Since this sentence is referring to the IDL analysis libraries and not the overall SSW "management and distribution" framework, this is an important distinction.

Reply:
Thanks @kreardon. Since I am not exactly sure what was meant by this IDL paragraph in the first place, I will let someone else submit a PR to fix these issues.

Reply:
fixed to `Since then, significant functionality was developed by the solar community and freely distributed as the SolarSoftWare library \citep{Freeland:1998we}.`